src/backend/codegen/include/codegen/utils/codegen_utils.h

```diff
@@ -249,8 +249,14 @@ class CodeGenerator {
         module_.get());
   }
 
+  template <typename FuncType>
+  llvm::Function* CreateFunctionTypeDef(
```

Comment:
Like we discussed, I will make default GetFunctionPointer to take typedef function type and change existing GetFuctionPointer to GetFunctionPointerImp private function